{{cookiecutter.repo_name}}/config/settings/production.py

```diff
@@ -118,7 +118,7 @@
 
 # URL that handles the media served from MEDIA_ROOT, used for managing
 # stored files.
-{% if cookiecutter.use_whitenoise == 'y' -%}
+{% if cookiecutter.use_whitenoise == "y" -%}
```

Comment:
When it comes to the cookiecutter template this project uses double quotes. I just changed these so it's consistent.